bf_downloads.html

```diff
@@ -123,6 +123,15 @@
                                 <td align="center">@MATLAB_TOOLS_SHA1@ (<a
                                         href="@MATLAB_TOOLS@.sha1">SHA1</a>)</td>
                             </tr>
+                            <tr>
+                                <td><a href="@OCTAVE_PACKAGE@"><img
+                                        src="images/bf_octave_package.png"
+                                        alt="Octave Package" /></a></td>
+                                <td align="center">@OCTAVE_PACKAGE_SIZE@</td>
+                                <td>@OCTAVE_PACKAGE_BASE@</td>
+                                <td align="center">OCTAVE_PACKAGE_SHA1@ (<a
```

Comment:
There is a missing `@` before `OCTAVE_PACKAGE_SHA1@` which prevents the token to be replaced